js/impress-audio.js

```diff
@@ -0,0 +1,77 @@
+/*
+https://github.com/danielsimons1/impress-audio
+Author: Daniel Simons
+Author Email: daniel.simons1@gmail.com
+Version: 1.0.0
+License: Free General Public License (GPL)
+*/
+(function(document, $, undefined) {
+  'use strict';
+
+  var isPlaying, waitTimeout ,
+    audio,video,media,currPlaying,
```

Comment:
Why are you mixing space style? In this line you have no spaces around ',' and in previous line you have.